lib/ddtrace/contrib/cucumber/patcher.rb

```diff
@@ -0,0 +1,54 @@
+require 'ddtrace/contrib/patcher'
+require 'ddtrace/ext/integration'
+require 'ddtrace/ext/ci'
+require 'ddtrace/ext/net'
+require 'ddtrace/contrib/analytics'
+
+module Datadog
+  module Contrib
+    module Cucumber
+      # Patcher enables patching of 'cucumber' module.
+      module Patcher
+        include Contrib::Patcher
+
+        module_function
+
+        def target_version
+          Integration.version
+        end
+
+        def patch
+          require 'ddtrace/pin'
+
+          Datadog::Pin.new(
+            Datadog.configuration[:cucumber][:service_name],
+            app: Datadog::Contrib::Cucumber::Ext::APP,
+            app_type: Datadog::Ext::AppTypes::TEST,
+            tags: Datadog::Ext::CI.tags(ENV).merge(Datadog.configuration.tags),
+            tracer: -> { Datadog.configuration[:cucumber][:tracer] }
+          ).onto(::Cucumber)
+
+          patch_cucumber_runtime
+        end
+
+        def patch_cucumber_runtime
+          require 'ddtrace/contrib/cucumber/formatter'
```

Comment:
Same question as https://github.com/DataDog/dd-trace-rb/pull/1216/files#r512077848 , can we move this to top level?